.travis.yml

```diff
@@ -6,16 +6,31 @@ php:
     - 5.5
     - 5.6
     - hhvm
+    - 7.0
 
 env:
     - WP_VERSION=latest WP_MULTISITE=0
     - WP_VERSION=latest WP_MULTISITE=1
-    - WP_VERSION=4.1 WP_MULTISITE=0
-    - WP_VERSION=4.1 WP_MULTISITE=1
-    - WP_VERSION=4.0 WP_MULTISITE=0
-    - WP_VERSION=4.0 WP_MULTISITE=1
-    - WP_VERSION=3.9 WP_MULTISITE=0
-    - WP_VERSION=3.9 WP_MULTISITE=1
+    # Current stable
+    - WP_VERSION=4.5 WP_MULTISITE=0
+    - WP_VERSION=4.5 WP_MULTISITE=1
+    # Previous Stable
+    - WP_VERSION=4.4.2 WP_MULTISITE=0
+    - WP_VERSION=4.4.2 WP_MULTISITE=1
+    # earliest supported that includes phpunit tests in core WP
+    - WP_VERSION=3.7 WP_MULTISITE=1
```

Comment:
Agreed. Last stable and also latest version is 4.5. Could bump that to 4.4.2 and then bump 4.4.2 down to 4.4.1 or 4.3.3?